contracts/libraries/RewardLib.sol

```diff
@@ -18,28 +18,14 @@ library RewardLib {
     int32 ratingToReward;
   }
 
-  struct StatusRewardContainer {
```

Comment:
Move `StatusRewardContainer` and `StatusReward` to separate lib to separate logic of main contract and token.